documentation/api/io.strimzi.api.kafka.model.EntityTopicOperatorSpec.adoc

```diff
@@ -40,6 +40,8 @@ spec:
         type: inline
         loggers:
           rootLogger.level: INFO
+          logger.io.strimzi.operator.topic.ZkTopicStore: TRACE
+          logger.io.strimzi.operator.topic.TopicOperator: DEBUG
```

Comment:
As Jakub said in another comment, this is not working for all log4j2 components. 

This is an example to change a package log level:

```sh
spec:
  entityOperator:
    topicOperator:
      logging:
        type: inline
        loggers:
          rootLogger.level: INFO
          logger.top.name: io.strimzi.operator.topic
          logger.top.level: DEBUG
```

This is an example to change a class log level:

```sh
spec:
  entityOperator:
    topicOperator:
      logging:
        type: inline
        loggers:
          rootLogger.level: INFO
          logger.toc.name: io.strimzi.operator.topic.TopicOperator
          logger.toc.level: DEBUG
```

It is easier for the existing/default loggers that you can find in cluster-operator/src/main/resources/default-logging:

```sh
spec:
  entityOperator:
    topicOperator:
      logging:
        loggers:
          logger.clients.level: DEBUG
```

At runtime, all the above configs are stored the following config map:

```sh
$ kubectl get cm my-cluster-entity-topic-operator-config -o yaml
apiVersion: v1
data:
  log4j2.properties: |
    # Do not change this generated file. Logging can be configured in the corresponding Kubernetes resource.
    name=TOConfig
    appender.console.type=Console
    appender.console.name=STDOUT
    appender.console.layout.type=PatternLayout
    appender.console.layout.pattern=%d{yyyy-MM-dd HH:mm:ss,nnnnn} %-5p [%t] %c{1}:%L - %m%n
    rootLogger.level=INFO
    rootLogger.appenderRefs=stdout
    rootLogger.appenderRef.console.ref=STDOUT
    rootLogger.additivity=false
    logger.clients.name=org.apache.kafka.clients
    logger.clients.level=DEBUG
    logger.clients.additivity=false
    logger.streams.name=org.INFO.kafka.streams
    logger.streams.level=INFO
    logger.streams.additivity=false
    logger.okhttp3.name=okhttp3
    logger.okhttp3.level=INFO
    logger.okhttp3.additivity=false
    logger.toc.level=INFO
    logger.toc.name=io.strimzi.operator.topic.TopicOperator

    monitorInterval=30
kind: ConfigMap
metadata:
  creationTimestamp: "2023-05-17T05:59:05Z"
  labels:
    app.kubernetes.io/instance: my-cluster
    app.kubernetes.io/managed-by: strimzi-cluster-operator
    app.kubernetes.io/name: entity-topic-operator
    app.kubernetes.io/part-of: strimzi-my-cluster
    strimzi.io/cluster: my-cluster
    strimzi.io/component-type: entity-topic-operator
    strimzi.io/kind: Kafka
    strimzi.io/name: my-cluster-entity-topic-operator
  name: my-cluster-entity-topic-operator-config
  namespace: test
  ownerReferences:
  - apiVersion: kafka.strimzi.io/v1beta2
    blockOwnerDeletion: false
    controller: false
    kind: Kafka
    name: my-cluster
    uid: e5793828-703f-4d9a-957d-fa7c0f36c4a4
  resourceVersion: "1252331"
  uid: faeb38c3-179f-4e12-8234-86eebc8e90c8
```

Reply:
Right. Okay. I'll use these examples and highlight the config for individual packages and classes